### DISCUSSION TITLE
Recommended way to build windows-based Docker images along with Linux images

### DISCUSSION THREAD

**Discussion by brainplot** [goreleaser/goreleaser#2440] [Q&A] [answered]
Hi, I'm trying to build my project using GoReleaser and build both linux-based and windows-based Docker images. I'm finding this a bit troublesome to achieve as I cannot run GoReleaser on Linux first and then Windows, in the GitHub Actions pipeline. That would result in GoReleaser trying to create the same GitHub release twice.

I thought about running a separate Windows pipeline just to get the Docker image built but that wouldn't update the GitHub release with the `docker pull ...` commands, like GoReleaser does, unless I try to somehow "manually" edit the Release with an ad-hoc action.

Is there a less error-prone solution?

**Comment by caarlos0** [comment, marked as answer]
I'm sorry, I don't have any experience with Docker on windows, so I can't weight in in that particularly, but I can say that right now you cant "merge" two goreleaser runs...

on pro, you [can import pre-existing binaries](https://carlosbecker.com/posts/goreleaser-prebuilt/), but if I understand your problem correctly, that won't help you...

**Reply by brainplot** [reply]
Thanks for your reply. Unfortunately I think I'll have to drop goreleaser from our CI. We really need those Windows images!
If anyone has a fitting solution, I'd be happy to hear it!

**Reply by caarlos0** [reply]
I understand, I hope you find a good solution 🙏
